src/hotspot/share/runtime/globals.hpp

```diff
@@ -1484,6 +1484,9 @@ const intx ObjectAlignmentInBytes = 8;
   product(ccstr, MetaspaceReclaimPolicy, "balanced",                        \
           "options: balanced, aggressive, none")                            \
                                                                             \
+  product(bool, PrintMetaspaceStatistics, false, DIAGNOSTIC,                \
```

Comment:
Can you call this PrintMetaspaceStatisticsAtExit to be like the other ones?

Reply:
Sure!